.github/workflows/build-git-installers.yml

```diff
@@ -413,3 +413,93 @@ jobs:
           name: deb-package-signed
           path: ${{ env.ARTIFACTS_DIR }}/signed
   # End build & sign Ubuntu package
+
+  create-github-release:
+    runs-on: ubuntu-latest
+    needs: [prereqs, windows_artifacts, mac_artifacts, ubuntu_sign-artifacts]
+    if: |
+      success() ||
+        (needs.ubuntu_sign-artifacts.result == 'skipped' &&
+        needs.mac_artifacts.result == 'success' &&
+        needs.windows_artifacts.result == 'success')
+    steps:
+      - name: Download Windows portable installer
+        uses: actions/download-artifact@v2
+        with:
+          name: win-portable-x86_64
+          path: win-portable-x86_64
+      - name: Download Windows x86_64 installer
+        uses: actions/download-artifact@v2
+        with:
+          name: win-installer-x86_64
+          path: win-installer-x86_64
```

Comment:
Action item for later: we have also been including the PDB files with our releases, on the off chance that we need to do a perf investigation with PerfView.

Reply:
The PDB files included in `microsoft/git` releases ([example](https://github.com/microsoft/git/releases/tag/v2.32.0.vfs.0.3)) differ from those included in `git-for-windows/git` releases ([example](https://github.com/git-for-windows/git/releases/tag/v2.32.0.windows.2)) - it looks like `git-for-windows/git` includes PDBs for external dependencies as well. Which set should be included here?

Reply:
I'll defer to @dscho here. I've only needed the PDBs for Git and not the rest. Further, our Azure Pipelines build only generated those PDBs. We should also be able to leverage the PDBs from `git-for-windows/git` if we ever need to go into those dependencies.

Reply:
If it's alright with you, I'll open a separate issue for this - minor changes are needed in `git-for-windows/build-extra` to include the PDBs in the installer (since this is built slightly differently than in the existing Azure DevOps pipeline) .

Reply:
That works for me. Incremental progress is important.